agg.pretty/RJMG16X.kicad_mod

```diff
@@ -0,0 +1,37 @@
+(module RJMG16X (layer F.Cu) (tedit 592324CE)
+  (descr http://www.farnell.com/datasheets/71713.pdf)
+  (fp_text reference REF** (at 0 -12.03) (layer F.Fab)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_text value RJMG16X (at 0 16.1) (layer F.Fab)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_line (start -8.4 -10.92) (end 8.4 -10.92) (layer F.SilkS) (width 0.15))
+  (fp_line (start 8.4 -10.92) (end 8.4 -4.45) (layer F.SilkS) (width 0.15))
+  (fp_line (start 8.4 14.78) (end -8.4 14.78) (layer F.SilkS) (width 0.15))
+  (fp_line (start -8.4 14.78) (end -8.4 -1.8) (layer F.SilkS) (width 0.15))
+  (fp_line (start -8.5 14.88) (end -8.5 -11.02) (layer F.Fab) (width 0.01))
+  (fp_line (start -8.4 -4.45) (end -8.4 -10.92) (layer F.SilkS) (width 0.15))
+  (fp_line (start 8.4 -1.8) (end 8.4 14.78) (layer F.SilkS) (width 0.15))
+  (fp_line (start -8.5 14.88) (end 8.5 14.88) (layer F.Fab) (width 0.01))
+  (fp_line (start 8.5 14.88) (end 8.5 -11.02) (layer F.Fab) (width 0.01))
```

Comment:
I think your silk line is showing the exact part outline and the fab lines are a sort of box around that, but the fab line should show the exact part dimensions and the silk live inside it instead.